model/user.go

```diff
@@ -2,17 +2,21 @@ package model
 
 import (
 	"github.com/jinzhu/gorm"
+	"github.com/nu7hatch/gouuid"
 	"godemo/database"
 	"golang.org/x/crypto/bcrypt"
 )
 
+// ユーザ情報を保持するモデル
```

Comment:
comment on exported type User should be of the form "User ..." (with optional leading article)